quetz/cli.py

```diff
@@ -483,5 +485,30 @@ def list() -> NoReturn:
         typer.echo('\n'.join([p for p in deployments]))
 
 
+@app.command()
+def plugin(cmd: str, path: str = typer.Argument(None, help="Path to the plugin folder")) -> NoReturn:
+
```

Comment:
it might be also cool to install directly from github:

```
quetz plugin install btel/quetz-plugin
```

Reply:
yeah that would be cool ...